ponder.schema.ts

```diff
@@ -1,23 +1,18 @@
-import { createSchema } from "@ponder/core";
+import { onchainTable } from "ponder";
 
-// Keeping the schema simple for now. We could have relationships as:
-// - new Cohort table with Cohort info (name, balance, totalWithdrawn, url, etc)
-// - new Builder table with Builder info (ens, etc)
-// - many-to-many relationship between Cohort and Builder (as CohortBuilder)
-export default createSchema((p) => ({
-  CohortBuilder: p.createTable({
-    id: p.string(),
-    amount: p.float(),
-    cohortContractAddress: p.bytes(),
-    timestamp: p.bigint(),
-    ens: p.string().optional(),
-  }),
-  CohortWithdrawal: p.createTable({
-    id: p.string(),
-    builder: p.bytes(),
-    amount: p.float(),
-    cohortContractAddress: p.bytes(),
-    reason: p.string(),
-    timestamp: p.bigint(),
-  }),
+export const cohortBuilder = onchainTable("cohort_builder", (t) => ({
+  id: t.text().primaryKey(),
+  amount: t.real().notNull(),
+  cohortContractAddress: t.hex().notNull(),
```

Comment:
Maybe you can create an index by cohortContractAddress so that the index is used when we get the builders from a cohort.

Reply:
Yup makes sense! Just created it at https://github.com/BuidlGuidl/bg-ponder-indexer/pull/3/commits/de706e96eb2d5f89a5c3be14f501595a4f69df9e